eng/scripts/Logging-Functions.ps1

```diff
@@ -10,6 +14,9 @@ function LogWarning {
   if (Test-SupportsDevOpsLogging) {
     Write-Host ("##vso[task.LogIssue type=warning;]$args" -replace "`n", "%0D%0A")
   }
+  elseif (Test-SupportsGitHubLogging) {
```

Comment:
I guess we could dedup all these `if/else` blocks into a function that smartly formats the message to fit ADO or GH?

Reply:
I agree the code could be improved a bit by refactoring, and won't object if you want to submit a PR, but I didn't think it was worth it.  It's still a pretty small file, and I don't see it growing much more.